.env.default

```diff
@@ -0,0 +1 @@
+IPFS_PATH=/opt/ipfs
```

Comment:
Is this used by the ipfs node? Don't you need to pass the options to the constructor?

Reply:
Yes, you're right. Forgot to add the constructor to use the env variable